docker/vm_boot_images/Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN apt-get install python3-tk libgl1-mesa-glx libxt-dev -y
 
 # Requirements for the tensorflow project
 RUN pip3 install --upgrade pip
-RUN pip3 install -r pre_requirements.txt
+#RUN pip3 install -r pre_requirements.txt
```

Comment:
Love the cleanup of duplicate packages! But this gets rid of `pre_requirements.txt` altoghether, right?

Reply:
Yes. Feel free to resurrect if necessary but tests pass on the cloud still.